blacksheep/server/application.py

```diff
@@ -608,3 +608,36 @@ async def __call__(self, scope, receive, send):
 
         request.scope = None  # type: ignore
         request.content.dispose()
+
+
+class ASGIApplication(Application):
```

Comment:
Also, not sure that's the best name for the class.

Reply:
I agree, I changed mind and don't like the name `ASGIApplication` - the `Application` class is already meant to be compatible with ASGI. Can you think of a better name? (I'll write something more in Gitter)